ci/pipelines/deploy-to-staging.yml

```diff
@@ -141,4 +168,80 @@ jobs:
         params:
           image: toolbox-ecr-registry-staging/image.tar
           additional_tags: toolbox-ecr-registry-staging/tag
-      
\ No newline at end of file
+
+  - name: deploy-frontend-to-staging
+    plan:
+      - get: frontend-ecr-registry-staging
+      - get: pay-infra
+      - get: pay-ci
+      - load_var: tag
+        file: frontend-ecr-registry-staging/tag
+      - task: assume-role
+        file: pay-ci/ci/tasks/assume-role.yml
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((pay_aws_staging_account_id)):role/concourse
+          AWS_ROLE_SESSION_NAME: terraform-staging-assume-role
+      - load_var: role
+        file: assume-role/assume-role.json
+        format: json
+      - task: deploy-to-staging
+        file: pay-ci/ci/tasks/deploy-app.yml
+        params:
+          APP_NAME: frontend
+          <<: *deploy_params
+      - task: wait-for-deploy
+        file: pay-ci/ci/tasks/wait-for-deploy.yml
+        params:
+          APP_NAME: frontend
+          <<: *wait_for_deploy_params
+
+  - name: smoke-test-frontend-on-staging
+    plan:
+      - get: frontend-ecr-registry-staging
+        trigger: true
+        passed: [deploy-frontend-to-staging]
+      - task: smoke-test-on-staging
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: govukpay/concourse-runner
+          run:
+            path: /bin/bash
+            args:
+            - -ec
+            - |
+              echo "Imagine we just smoked on staging."
+  - name: frontend-pact-tag
+    plan:
+      - get: frontend-ecr-registry-staging
+        passed: [smoke-test-frontend-on-staging]
+        trigger: true
+      - load_var: tag
+        file: frontend-ecr-registry-staging/tag
+      - get: pay-ci
+      - task: get-git-sha-for-release-tag
+        file: pay-ci/ci/tasks/get-git-sha-for-release-tag.yml
+        params:
+          APP_NAME: frontend
+          TAG: ((.:tag))
+      - load_var: git-sha
+        file: git-sha/git-sha
+      - task: tag-pact
+        file: pay-ci/ci/tasks/pact-tag.yml
+        params:
+          GIT_SHA: ((.:git-sha))
+          APP_NAME: frontend
+          PACT_TAG: staging-fargate
+  - name: push-frontend-to-production-ecr
+    plan:
+      - get: frontend-ecr-registry-staging
+        params:
+          format: oci
+        trigger: true
+        passed: [smoke-test-frontend-on-staging]
```

Comment:
I don't have an issue with this but probably best to make it `[frontend-pact-tag]` to keep it consistent with what we do in test.

Reply:
👍